localization/en.ftl

```diff
@@ -246,3 +246,17 @@ custom-mathquill-config-opt-leftIntoSubscript-name = Left/Right into Subscripts
 custom-mathquill-config-opt-leftIntoSubscript-desc = Moving the cursor left or right will go into subscript instead of superscript
 custom-mathquill-config-opt-extendedGreek-name = More Greek Letters
 custom-mathquill-config-opt-extendedGreek-desc = Enables replacements for all supported greek letters
+
+## Syntax highlightAlternatingLines
+syntax-highlighting-name = Syntax Highlighting
+syntax-highlighting-desc = Color in various parts of expressions to make them easier to reason about.
+syntax-highlighting-opt-bracketPairColorization-name = Bracket Pair Colorization
+syntax-highlighting-opt-bracketPairColorization-desc = Applies a set of alternating colors to brackets (e.g. ()[]{}||) to make matching bracket pairs easy to spot.
```

Comment:
`{}` is ftl syntax, so "Error formatting key syntax-highlighting-opt-bracketPairColorization-desc".  I think you fix it with `{"{}"}`

Reply:
fixed